manuscript.md

```diff
@@ -33,10 +33,39 @@ Yet another of the issues pertains to
 
 ### Network structure
 
-<!-- Most important right now -->
+Research outputs are typically papers, which report on at least one research cycle after it has occurred. The communicative design of papers embeds hindsight and its biases in the reporting of results. Moreover, this design eliminates the verification of the chronology within a paper. On the other hand, the unit of a paper is encompasses so much that citations to other papers can indicate a tangent or a crucial link. Additionally, the paper is a bottleneck for what is communicated: It cannot properly deal with code, data, materials, etc.
```

Comment:
unit of a paper 'is' (remove 'is')

Reply:
Needs concluding sentence; something like: "Therefore, the paper as research output limits the effectiveness of scientific communication.